projects/RPi/devices/Slice/patches/kodi/kodi-004-keyboard.patch

```diff
@@ -0,0 +1,22 @@
+--- a/system/keymaps/keyboard.xml	2016-09-17 16:35:20.000000000 +0100
++++ b/system/keymaps/keyboard.xml	2016-10-01 19:31:07.928719606 +0100
+@@ -56,7 +56,7 @@
+       <menu mod="longpress">Menu</menu>
+       <c>ContextMenu</c>
+       <c mod="longpress">Menu</c>
+-      <space>Pause</space>
++      <space>PlayPause</space>
+       <x>Stop</x>
+       <period>SkipNext</period>
+       <comma>SkipPrevious</comma>
+@@ -321,8 +321,8 @@
+       <l>NextSubtitle</l>
+       <left>StepBack</left>
+       <right>StepForward</right>
+-      <up>ChapterOrBigStepForward</up>
```

Comment:
This is something that I deliberately changed because without it you cannot control the volume from the Slice remote. I'm used to the AppleTV remote which uses up/down for volume and I think this is a better user experience. If you do the big-step stuff with up down I am forced to use two remotes ALL THE TIME. With this change I only need one remote; if the AVR is set to a decent average volume I only _occasionally_ need to tweak it when playing something that uses passthrough and comes out too loud/quiet.